app/controllers/idv/sessions_controller.rb

```diff
@@ -31,7 +44,7 @@ def start_idv_session
     end
 
     def applicant_from_params
-      app_vars = applicant_params.delete_if { |_key, value| value.blank? }
+      app_vars = idv_params.merge(financial_params.delete_if { |_key, value| value.blank? })
```

Comment:
Could you please explain the reasoning behind extracting the financial_params out of the applicant_params? It seems like the only place the `financial_params` are used is here, to merge them back again with the applicant params.